client/src/pages/ContactUsPage/ContactUsPage.js

```diff
@@ -0,0 +1,122 @@
+import React from 'react';
+import { Typography, Box, Grid, Link, Stack, styled } from '@mui/material';
+import FacebookIcon from '@mui/icons-material/Facebook';
+import TwitterIcon from '@mui/icons-material/Twitter';
+import LinkedInIcon from '@mui/icons-material/LinkedIn';
+import YouTubeIcon from '@mui/icons-material/YouTube';
+import PinterestIcon from '@mui/icons-material/Pinterest';
+import TelegramIcon from '@mui/icons-material/Telegram';
+import ContactsForm from '../../features/contactUs/components';
+
+const icons = [
+  <FacebookIcon />,
+  <TwitterIcon />,
+  <LinkedInIcon />,
+  <YouTubeIcon />,
+  <PinterestIcon />,
+  <TelegramIcon />,
+];
+
+const SocialsContainer = styled(Box)(() => ({
```

Comment:
you can use `Stack` component instead with `direction="row"`

Reply:
done in new commit